patches/codium-insiders.desktop.patch

```diff
@@ -0,0 +1,21 @@
+--- a/usr/share/applications/codium-insiders.desktop.orig	2023-12-22 14:10:13.462383894 -0300
++++ b/usr/share/applications/codium-insiders.desktop	2023-12-22 14:38:46.330986314 -0300
```

Comment:
this won't work. The patch is applied before the build start.

Reply:
Like I mention here, https://github.com/flathub/flathub/pull/4620#discussion_r1439898693